container/features/src/main/resources/features.xml

```diff
@@ -1883,4 +1883,7 @@
         <feature version="${cxfVersion}">cxf-jaxrs</feature>
         <bundle>mvn:org.opennms.core.health/org.opennms.core.health.rest-cxf/${project.version}</bundle>
     </feature>
+    <feature name="opennms-ui-extension" version="${project.version}" description="OpenNMS :: Features :: UI Extension">
+        <bundle>mvn:org.opennms.features/ui-extension/${project.version}</bundle>
```

Comment:
We could add this bundle to the existing `opennms-api-layer` feature instead of defining a new one.